src/sentry/services/hybrid_cloud/organization/service.py

```diff
@@ -59,32 +59,6 @@ def get_org_by_slug(
         """
         pass
 
-    # TODO: This should return RpcOrganizationSummary objects, since we cannot realistically span out requests and
-    #  capture full org objects / teams / permissions.  But we can gather basic summary data from the control silo.
-    @regional_rpc_method(resolve=UnimplementedRegionResolution())
-    @abstractmethod
-    def get_organizations(
-        self,
-        *,
-        user_id: Optional[int],
-        scope: Optional[str],
-        only_visible: bool,
-        organization_ids: Optional[List[int]] = None,
-    ) -> List[RpcOrganizationSummary]:
```

Comment:
Deleting this temporarily to make sure nothing breaks. I'll revert dbadfc9 before deploying, for compatibility with `getsentry`. See https://github.com/getsentry/getsentry/pull/10594